src/assets/environment.json

```diff
@@ -24,6 +24,7 @@
       "sourceRepository": "https://github.com/QutEcoacoustics/workbench-client",
       "sourceRepositoryIssues": "https://github.com/QutEcoacoustics/workbench-client/issues"
     },
+    "hideProjects": true,
```

Comment:
is there a reason for this default?

Reply:
Mostly leftover from testing, this will be changed to `false` on completion assuming i remember. Only a dev environment setting so it shouldnt matter too much